snyk-monitor/templates/pvc.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.pvc }}
```

Comment:
Just don't forget to be consistent with the previous @ivanstanev comment `.Values.pvc.enabled`

Reply:
Good spot, thanks!